Gemfile

```
@@ -6,15 +6,17 @@ gem "activesupport", require: false
 gem "mry", require: false
 gem "parser"
 gem "pry", require: false
-gem "rubocop", "0.92.0", require: false
-gem "rubocop-i18n", require: false
-gem "rubocop-migrations", require: false
+gem "rubocop", "1.4.1", require: false
+gem "rubocop-i18n", require: false,
+  github: "mvz/rubocop-i18n", ref: "5e824a47a88732830a8b60e46697bbb0a2795c19"
+gem "rubocop-migrations", require: false,
+  github: "DanielWright/rubocop-migrations", branch: "dw/nov24/rubocop-1.x-support"
```

```suggestion
gem "rubocop-migrations", require: false
```
This change should be applied when https://github.com/redbooth/rubocop-migrations/pull/3 is merged and released.